src/stores/options.json

```diff
@@ -158,6 +158,16 @@
         "shortDescription": "The profile has sent a telemetry 'core' ping from Lockwise Android on the day in question",
         "disabledDimensions": ["os", "channel", "attributed"]
       },
+      {
+        "label": "Any Lockwise Baseline Activity",
+        "key": "Any Lockwise Baseline Activity",
+        "shortDescription": "The profile has sent a Glean 'baseline' ping from Lockwise on the day in question",
+        "disabledDimensions": ["channel", "attributed"],
+        "platforms": [
```

Comment:
Is this the right way to specify a custom set of OS values?

Reply:
There is no way to do this currently, but it's clear we should move toward a model where this is easier.

Reply:
For the time being, I can separate this out on the backend to two separate usage criteria for "Any Lockwise Android Baseline Activity" and "Any Lockwise iOS Baseline Activity" so we can continue to ignore "os".